static/styles/index.scss

```diff
@@ -112,16 +123,18 @@
     &__btns {
       height: 20%;
       display: flex;
-      justify-content: space-around;
+      justify-content: space-between;
     }
   }
 
   .footer {
     grid-column: 1/3;
     grid-row: 3/4;
   }
-
-  .rules-icon {
-    font-size: 1.2em;
+  
+  .halloffame-icon {
```

Comment:
Perhaps you could change this class name to more BEM-like

Reply:
> Perhaps you could change this class name to more BEM-like

Ok, we will work on it ;)